components/data-table/index.jsx

```diff
@@ -371,6 +455,29 @@ class DataTable extends React.Component {
 				</tbody>
 			</table>
 		);
+
+		if (this.props.fixedHeader) {
+			return (
```

Comment:
This is a little nit-picky, but remove double return exits from the function to increase readability and use something like a `let` instead. I'm open to ternary return, too.